code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -52,6 +52,19 @@
 	var/warning_low_pressure = WARNING_LOW_PRESSURE   // Low pressure warning.
 	var/hazard_low_pressure = HAZARD_LOW_PRESSURE     // Dangerously low pressure.
 
+	var/list/atmos_requirements = list(
+		"min_oxy" = 16,
+		"max_oxy" = INFINITY,
```

Comment:
didn't i literally leave a line comment saying "0 is nothing because i don't like INFINITY"